src/components/RecipeCreateForm.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { TextField, Button, Alert } from "@mui/material";
+import Recipes from "../modules/Recipes";
+
+const RecipeCreateForm = () => {
+  const [recipe, setRecipe] = useState({});
+  const [message, setMessage] = useState();
+
+  const createRecipe = async () => {
+    const response = await Recipes.create(recipe);
+    setMessage(response.data.message);
```

Comment:
When you return the "response.data" from the "Recipes.js", here you just need to set the message like:
```
setMessage(response.message)
```
This way you'll display either an error message or a successful message.